src/closure-externs.js

```diff
@@ -1100,3 +1100,10 @@ var GL;
  * @suppress {undefinedVars}
  */
 var SDL;
+
+// Module loaders externs, primarily for Binaryen
```

Comment:
If this is primarily for binaryen, could this be done there instead?

Reply:
I believe it is not possible pass additional arguments like externs to Closure Compiler externally, at least not right now. Also Binaryen is tightly related project.

However, allowing external arguments to Closure Compiler would be useful in general if someone can implement it.

Reply:
Thanks, I see.

Let's leave this as it is, then. Also I think `define` is from AMD, so it could help other things too, so makes sense to do it here.

lgtm, just waiting to merge on your ping to @dcodeIO.

Reply:
Makes sense to me to have `define` being declared. For instance, `wabt.js` has a very similar wrapper that will benefit from this.

Ideally, binaryen.js would have its own specific externs. That would also remove the necessity to use `[' ... ']` everywhere to prevent renaming. But that's another story, of course.